pkg/option/config.go

```diff
@@ -3060,7 +3049,6 @@ func (c *DaemonConfig) Populate(logger *slog.Logger, vp *viper.Viper) {
 		c.IdentityRestoreGracePeriod = defaults.IdentityRestoreGracePeriodKvstore
 	}
 
-	c.LoadBalancerProtocolDifferentiation = vp.GetBool(LoadBalancerProtocolDifferentiation)
```

Comment:
You'll probably have to regenerate the documentation following this.

Reply:
This is probably not required (CI doesn't seem to complain) because the flag was already marked as deprecated.